.circleci/config.yml

```diff
@@ -350,7 +350,9 @@ commands:
           name: Install cargo deny, about, edit
           command: |
             if [[ ! -f "$HOME/.cargo/bin/cargo-deny$EXECUTABLE_SUFFIX" ]]; then
-              cargo install --locked cargo-deny cargo-about cargo-edit
+              cargo install --locked --version 0.14.21 cargo-deny
```

Comment:
Silly question, but is there  a way to make this _also_ locked for the person running the tool locally? e.g., a project-specific constraint on the binary toolchain used?

Reply:
There's a crate we could use in xtask: https://crates.io/crates/cargo-local-install
But I have not had time to look into it yet.
I suspect the regular changes to licenses.html are due to different versions of cargo-about installed locally